Plugins~/Src/MeshSyncClientBlender/msblenContext.cpp

```diff
@@ -1448,10 +1449,19 @@ bool msblenContext::sendObjects(MeshSyncClient::ObjectScope scope, bool dirty_al
     }
 
 #if BLENDER_VERSION >= 300
-    if (m_geometryNodeUtils.getInstancesDirty() || dirty_all) {
+    // The dependency graph update event does not fire when the scene changes while the timeline is playing.
+    // To ensure geometry nodes get exported correctly while playing, check the frame number:
+    static int lastExportedFrame = scene.GetCurrentFrame();
+    int currentFrame = scene.GetCurrentFrame();
```

Comment:
Just a concern for the future with this paradigm:
what is the lifetime of lastExportedFrame?
If we start writing unit tests, would we have to reset the variable before each tests? how would this work?

Reply:
Lifetime is for the duration of the program. A unit test would need to take this into account and treat it like a meshsync session where frame updates trigger an export. To reset the frame variable, it could set the scene frame and call this.

Reply:
> To reset the frame variable, it could set the scene frame and call this
Would also require to force it somehow to get through the if statement bellow. 

Would the var need to be reset as well when users open a new project? 
Might be better we have a small class that deals with this logic, holds an internal state with the counters and can be reset on an event on the blender context?

Reply:
The first sync would be a full sync and sync instances anyway so it's not a problem if the first value is not the current frame.

Reply:
> A unit test would need to take this into account and treat it like a meshsync session where frame updates trigger an export. To reset the frame variable, it could set the scene frame and call this.

This would block us from running tests in parallel. I don't mind the particular example here, and so I approved this PR, but I am more concerned about repeating this pattern across the codebase.